src/store/fileUploadStore.js

```diff
@@ -245,10 +246,21 @@ const actions = {
 				// Mark the file as uploaded in the store
 				commit('markFileAsSuccessUpload', { uploadId, index, sharePath })
 			} catch (exception) {
-				console.debug('Error while uploading file:' + exception)
-				showError(t('spreed', 'Error while uploading file'))
+				console.error(`Error while uploading file "${fileName}":` + exception, fileName, exception.response.status)
+				const temporaryMessage = state.uploads[uploadId].files[index].temporaryMessage
+				let reason = 'failed-upload'
+				if (exception.response.status === 507) {
+					reason = 'quota'
+					showError(t('spreed', 'Error while uploading file "{fileName}": insufficient storage', { fileName }))
```

Comment:
File app says "Not enough free space" maybe we can align with that?

Reply:
Wording aligned